app/jobs/assignment_repo/create_github_repository_job.rb

```diff
@@ -5,10 +5,17 @@ class CreateGitHubRepositoryJob < ApplicationJob
     queue_as :create_repository
     retry_on Creator::Result::Error, wait: :exponentially_longer, queue: :create_repository
 
-    # Create an AssignmentRepo.
+    class Status
+      CREATE_REPO = "Creating repository"
+      ADDING_COLLABORATOR = "Adding collaborator"
+      IMPORT_STARTER_CODE = "Importing starter code"
```

Comment:
The idea behind these statuses was to display something to the user as the repo creation job proceeds. 👍 👎 ❓

Reply:
If this class has nothing but these status messages, we can just define them on the job class. `CREATE_REPO_STATUS = "...`